apptest_test.go

```diff
@@ -55,7 +55,10 @@ func TestTest(t *testing.T) {
 		So(err, ShouldBeNil)
 	})
 
+	d := h.DNAPath()
 	Convey("it should fail the test on incorrect input types", t, func() {
+		Debugf("path: %v\n", d)
+		execCmd("ls", "-ltrR", d)
```

Comment:
Debug code that will be removed when all is working. I added execCmd to make it easier to run a command to see what is in the test dirs at different test points. (in test.go)